src/shared_modules/utils/registryHelper.h

```diff
@@ -199,6 +201,29 @@ namespace Utils
                 return ret;
             }
 
+            std::string keyModificationDate() const
+            {
+                std::string ret { };
```

Comment:
std::string don't need initialization, the constructor during instantiation does.